src/ebmc/show_properties.cpp

```diff
@@ -29,36 +29,37 @@ void ebmc_baset::show_properties()
 {
   unsigned p_nr=1;
 
+  auto make_xml =
+    [](const ebmc_propertiest::propertyt &p, std::size_t p_nr) -> xmlt {
+    xmlt xml("property");
+    xml.set_attribute("name", id2string(p.name));
+
+    xml.new_element("number").data = std::to_string(p_nr); // will go away
```

Comment:
Just curious: what will go away and when?

Reply:
That's just a moved comment, not new. You can't use the number for anything (we use the property identifier instead), so it's really redundant.